broker-cli/commands/wallet.js

```diff
@@ -531,6 +532,46 @@ async function unlock (args, opts, logger) {
   }
 }
 
+/**
+ * Returns all on-chain history of a specific wallet
+ * @param {Object} args
+ * @param {string} args.symbol
+ * @param {Object} opts
+ * @param {string} opts.rpcAddress
+ * @param {Logger} logger
+ * @returns {void}
+ */
+async function history (args, opts, logger) {
+  const { symbol } = args
+  const { rpcAddress } = opts
+
+  try {
+    const client = new BrokerDaemonClient(rpcAddress)
+    const { transactions = [] } = await client.walletService.walletHistory({ symbol })
+
+    if (!transactions) {
+      logger.info(`No summary available for ${symbol}`)
```

Comment:
This would almost be better in the table if the transactions length was zero so that you could have visual confirmation that there are not transactions (vs thinking it is a bug)